src/test/java/org/helpberkeley/memberdata/ControlBlockTestBase.java

```diff
@@ -241,24 +240,24 @@ public void opsManagerUnknownUserTest() {
                 MessageFormat.format(ControlBlock.UNKNOWN_OPS_MANAGER, "UnknownDudette"));
     }
 
-//    @Test
-//    public void opsManagerEmptyPhoneTest() {
-//
-//        String key = "OpsManager (UserName|Phone)";
-//        String value = "biff|";
-//
-//        String workFlowData = getHeader()
-//                + getBeginRow()
-//                + getVersionRow()
-//                + getKeyValueRow(key, value)
-//                + getEndRow();
-//
-//        WorkflowParser workflowParser = WorkflowParser.create(Map.of(), workFlowData);
-//
-//        Throwable thrown = catchThrowable(workflowParser::drivers);
-//        assertThat(thrown).isInstanceOf(MemberDataException.class);
-//        assertThat(thrown).hasMessageContaining("Empty OpsManager phone number at line 4. Does not match \"username\" or \"username | phone\"");
-//    }
+    @Test
+    public void opsManagerEmptyPhoneTest() {
+
+        String key = "OpsManager (UserName|Phone)";
+        String value = "biff|";
+
+        String workFlowData = getHeader()
+                + getBeginRow()
+                + getVersionRow()
+                + getKeyValueRow(key, value)
+                + getEndRow();
+
+        WorkflowParser workflowParser = WorkflowParser.create(Map.of(), workFlowData);
+
+        Throwable thrown = catchThrowable(workflowParser::drivers);
+        assertThat(thrown).isInstanceOf(MemberDataException.class);
+        assertThat(thrown).hasMessageContaining(MessageFormat.format(ControlBlock.OPS_MANAGER_WRONG_FORMAT, value, "4"));
```

Comment:
Just a note here about the line number parameter to the OPS_MANAGER_WRONG_FORMAT.  The production code is passing a long value and here we are passing a string ("4").  The output of MessageFormat.format() remains the same (it generates a single string).  MessageFormat.format() handles various explicit and implicit type handling/conversion rules.  See the [FormatType and FormatStyle explanations and table for the MessageFormat class](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/text/MessageFormat.html)